### PR TITLE
feat: http template for Quarkus stack

### LINKED ISSUE #31: quarkus http template
We currently have the cloudevents template ("events"), and would like to add support for an http handler as well.

### PR DESCRIPTION
**Pull request by matejvasek**
resolves #31

### CODE REVIEW

**Comment by matejvasek** [comment]
@lkingland note that this won't work yet, since buildpack is to picky and won't build `funqy-http` (it accepts only `funqy-knative-events`).

**Comment by lkingland** [comment]
Gotcha. Thanks for the note. We can at least have the template writing part working with that _caviat emptor_

**Comment by matejvasek** [comment]
Now that updated buildpacks have been released this should work.

File: templates/quarkus/http/.vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        },
+        {
+            "label": "Appsody: stop",
+            "type": "shell",
+            "command": "appsody stop",
+            "group": "build",
+            "problemMatcher": []
+        }
+    ]
+}
```
Comment (lance) [inline templates/quarkus/http/.vscode/tasks.json:47]: Nit: add newline

**Pull request merged**